examples/add.c

```diff
@@ -15,6 +15,18 @@
 #include "common.h"
 #include <assert.h>
 
+/**
+ * The following example demonstrates how to add files with libgit2.
+ *
+ * It will use the repository in the current working directory, and act
+ * on files passed as its parameters.
+ *
+ * Recognized options are:
+ *   -v/--verbose: show the file's status after acting on it.
+ *   -n/--dry-run: do not actually change the index.
+ *   -u/--update: update the index instead of adding to it.
+ */
```

Comment:
I'd kindly ask you to drop these from this PR until #4956 is merged. Otherwise I'm afraid that it will result in some non-trivial merge conflicts. I can cherry-pick these into #4956 if you want, or you can propose them at a later point, whatever works for you.

Reply:
I'll do that. I have at least 3 example-touching PR already, and I'd feel better handling them as part of #4956 if that's ok with you.

Reply:
Fair enough. I've cherry-picked your commit

Reply:
Thanks !